### DISCUSSION TITLE
Outreachy: welcoming new contributors to open source

### DISCUSSION THREAD

**Discussion by thibaudcolas** [wagtail/wagtail#9297] [Announcements]
Following on from [Wagtail’s participation](https://wagtail.org/blog/wagtail-cms-projects-for-google-summer-of-code-2022/) in Google Summer of Code, we’re now [joining Outreachy](https://wagtail.org/blog/outreachy-welcoming-new-contributors-to-open-source/) for another three months of open source internships! :)

Starting this discussion thread so people can share their thoughts, and for any questions about our involvement.

**Comment by p0lygun** [comment]
[joining Outreachy](https://wagtail.org/blog/wagtail-cms-projects-for-google-summer-of-code-2022/) this link goes to the GSOC blog ?
and if I understand this correctly this is for May 2023 internships? as the application phase for dec is already over

**Reply by thibaudcolas** [reply]
Arh :) thank you, I mistakenly pasted the same link twice. Here is the correct link: https://wagtail.org/blog/outreachy-welcoming-new-contributors-to-open-source/. This is for December 2022, so applications are indeed closed for this time around – but if there’s interest for future cohorts please do mention it.

**Comment by akotosheila28** [comment]
Hello @thibaudcolas,  I'm Sheila Akoto (she\her) an Outreachy applicant. I'm new here and would be delighted if you could guide me on how to contribute.

**Comment by sylwater** [comment]
I'm James Aghaji, an outreachy contributor. I'm proficient in Python3, Bash/shell scripting and research writing, among other things. Nice again to be on the team.
